compute/cloud-client/src/test/java/compute/disks/DisksIT.java

```diff
@@ -353,4 +363,15 @@ public void testCreateDiskSecondaryZonal()
     assertThat(status).isEqualTo(Status.DONE);
     assertEquals(SECONDARY_DISK, disk.getName());
   }
+
+  @Test
+  void testCreateDiskWithSnapshotSchedule()
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    Status status = CreateDiskWithSnapshotSchedule.createDiskWithSnapshotSchedule(
+            PROJECT_ID, ZONE, DISK_WITH_SNAPSHOT_SCHEDULE, SNAPSHOT_SCHEDULE);
+    Disk disk = Util.getDisk(PROJECT_ID, ZONE, DISK_WITH_SNAPSHOT_SCHEDULE);
+
+    assertThat(status).isEqualTo(Status.DONE);
+    assertEquals(DISK_WITH_SNAPSHOT_SCHEDULE, disk.getName());
```

Comment:
This assertion basically checks that the getDisk returns the correct disk. It does not verify if the sample.

I'd replace this assertion to verify that the Util.getDisk doesn't throw an exception when retrieving the disk.

Reply:
Thank you! Fixed